components/mercure.rst

```diff
@@ -0,0 +1,48 @@
+.. index::
+   single: Mercure
+   single: Components; Mercure
+
+The Mercure Component
+=====================
+
+    `Mercure`_ is an open protocol allowing to push data updates to web
+    browsers and other HTTP clients in a convenient, fast, reliable
+    and battery-friendly way.
+    It is especially useful to publish real-time updates of resources served
+    through web APIs, to reactive web and mobile apps.
+
+The Mercure Component implements the "publisher" part of the Mercure Protocol.
+
+Installation
+------------
+
+.. code-block:: terminal
+
+    $ composer require symfony/mercure
+
+Alternatively, you can clone the `<https://github.com/symfony/mercure>`_ repository.
+
+.. include:: /components/require_autoload.rst.inc
+
+Usage
+-----
+
+The following example shows the component in action::
+
+    // change these values accordingly to your hub installation
+    define('HUB_URL', 'https://demo.mercure.rocks/hub');
+    define('JWT', 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJtZXJjdXJlIjp7InN1YnNjcmliZSI6WyJmb28iLCJiYXIiXSwicHVibGlzaCI6WyJmb28iXX19.LRLvirgONK13JgacQ_VbcjySbVhkSmHy3IznH3tA9PM');
+
+    use Symfony\Component\Mercure\Publisher;
+    use Symfony\Component\Mercure\Update;
+    use Symfony\Component\Mercure\Jwt\StaticJwtProvide;
+
+    $publisher = new Publisher(URL, new StaticJwtProvide(JWT));
```

Comment:
URL -> HUB_URL ?

Reply:
It doesn't bring any value here IMO (see https://dave.cheney.net/practical-go/presentations/qcon-china.html#_choose_identifiers_for_clarity_not_brevity).

Reply:
@dunglas It's a mismatch with what's defined above... :see_no_evil:

Reply:
Got it, thanks

Reply:
sorry for being not clear

Reply:
@OskarStark it's me who is sorry, I wasn't attentive enough.